apps/metric_vnode/src/metric_io.erl

```diff
@@ -63,6 +65,9 @@ swrite(Pid, Bucket, Metric, Time, Value) ->
 read(Pid, Bucket, Metric, Time, Count, ReqID, Sender) ->
     gen_server:cast(Pid, {read, Bucket, Metric, Time, Count, ReqID, Sender}).
 
+read_rest(Pid, Bucket, Metric, Time, Count, Part, ReqID, Sender) ->
+    gen_server:cast(Pid, {read_rest, Bucket, Metric, Time, Count, Part, ReqID, Sender}).
```

Comment:
Line 69 is too long:     gen_server:cast(Pid, {read_rest, Bucket, Metric, Time, Count, Part, ReqID, Sender})..